sass/elements/_kbd.scss

```diff
@@ -0,0 +1,12 @@
+kbd {
+  font-size: 0.8rem;
+  padding-left: 0.1rem;
+  padding-right: 0.1rem;
+  padding-top: 0.1rem;
+  padding-bottom: 0.1rem;
+  line-height: 1;
+  border-color: grey;
+  border-radius: 0.3rem;
```

Comment:
should use `$border-radius` (defined in _vars.scss)

Reply:
Should it be done for migration tags too then?

Reply:
actually on second thought that's way too much radius. maybe `calc($border-radius / 2)`

Reply:
> Should it be done for migration tags too then?

oh, probably. that can be a separate MR.

Reply:
I checked all of that and it seems that `$border-radius` is used only for cards and menus, but not for anything else. In the tree it's even overridden to be `4px`. Therefore, I'm not sure that it should be calculated.

A possible solution for that issue is making additional variables to be used instead of primary and which might use `$border-radius`, but might not.